implementation.md

```diff
@@ -1554,3 +1554,268 @@ In the `Video` and `Audio` objects, sellers must only include one form of floor
+                        "id": "fac13741-0648-436a-88cf-aceafdf45c9a",
+			"atype": "1"
+                    }
+                ]
+            }
+        ]
+    }
+}
+```
+Note that there is no <code>buyeruid</code> in the example above. If the SSP does not have an explicit agreement from the DSP that they want to receive matched IDs in the <code>buyeruid</code> field, they would simply forward the above request to the DSP (either removing the eids array if the DSP doesn’t want to receive IDs from graphvendor.com, or retaining the array if the DSP does want to receive IDs from graphvendor.com in eids). 
+
+##### Example 4: Request from SSP → DSP corresponding to the above request from pub, where the SSP has set the DSP’s buyeruid based on a hosted cookie match table and the bridged ID they received from the pub (based on prior arrangement with the DSP only)
+
+If an arrangement exists to put a matched DSP ID in <code>buyeruid</code>, that would be done as follows:
+
+<b>NOTE:</b> This method should be used ONLY in cases where a previous arrangement has been made directly between the sell-side and the DSP to express a matched ID in <code>buyeruid</code>. If no agreement is in place, the ids in question should be sent in the EIDS array. 
+
+```
+{
+"user": {
+    "buyeruid": "3d89748d-74bb-44f1-9908-298090dc2944",
+    "eids": [
+        {
+            "matcher": "graphvendor.com",
+            "inserter": "pub.com",
+            "source": "sspdomain.com”,
+            "mm": 3,
+            "uids": [
+                {
+                    "id": "fac13741-0648-436a-88cf-aceafdf45c9a"
+		    "atype": "1"
+                }
+            ]
+        },
+        {
+            "inserter": "sspdomain.com",
+            "source": "dspdomain.com”,
+            "mm": 3,
+            "uids": [
+                {
+                    "id": "3d89748d-74bb-44f1-9908-298090dc2944"
+                }
+            ]
+        }
+    ]
+}
+}
+```
+note that <code>mm</code>:3 is retained in this example, because the match is built on top of an ID that was itself based on cross-domain authentication (e.g. via hashed email)
+
+#### 7.12.4.3 Derrived IDs Across Multiple Properties or Devices
+##### Example 5: SSP matches their own ID to an email-based hash to provide a universal ID to the DSP in the buyeruid field (based on prior arrangement to put it in buyeruid)
+```
+{
+"user": {
+    "buyeruid": "fac13741-0648-436a-88cf-aceafdf45c9a",
+    "eids": [
+        {
+            "source": "sspdomain.com”,
+            "mm": 1,
+            "uids": [
+                {
+                    "id": "3d89748d-74bb-44f1-9908-298090dc2944"
+		    "atype": "3"
+                }
+            ]
+        },
+        {
+            "matcher": "idvendor.com",
+	    "inserter": "sspdomain.com",
+            "source": "universalid.com”,
+            "mm": 4,
+            "uids": [
+                {
+                    "id": "fac13741-0648-436a-88cf-aceafdf45c9a"
```

Comment:
this is missing "atype", and I believe it should be "atype":3

Reply:
added